.travis.yml

```diff
@@ -1,20 +1,37 @@
 language: python
+
+env:
+  - MOZ_HEADLESS=1
+
+addons:
+  firefox: "55.0"
+
 python:
   - "2.7"
+
 services: postgresql
+
+before_install:
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.20.1/geckodriver-v0.20.1-linux64.tar.gz
+  - tar -xzvf geckodriver-v0.20.1-linux64.tar.gz
+  - sudo mv geckodriver /usr/local/bin
+
 install:
   - pip install -r requirements.txt
+
 before_script:
   - psql -c "create role vmsadmin with createrole createdb login password '0xdeadbeef';" -U postgres
   - psql -c "CREATE DATABASE vms;" -U postgres
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3
   - export DJANGO_SECRET_KEY=foobarbaz
+
 script:
   - cd vms
   - python manage.py syncdb --noinput
   - python manage.py migrate --noinput --traceback --settings=vms.settings
-  - coverage run --source='.' manage.py test
+  - coverage run --source='.' manage.py test ../tests/
```

Comment:
why `test ../tests/ ?` also, please add a `-v 2` so you easily know which test failed.

Reply:
Earlier we did `cd vms` so we are not in root but inside vms dir.

Reply:
I'll check if `coverage` check verbosity levels.
I had one thing in mind that we can do `coverage report -m` to better report the coverage in travis log itself.

Reply:
> Earlier in travis we did cd vms so we are not in root but inside vms dir.

the current PR has that as well  in line 31 ?

Reply:
yes line 31 to be precise.

Reply:
okey - so I need to make my point clear. if you are at manage.py level (the BASE_DIR) place, why are these tests one level up ? its a bit weird tbh.

Reply:
Okay I checked and tested it also. coverage supports verbosity levels.
Check this build for further assurance - [test-build](https://travis-ci.org/Monal5031/vms/builds/381437343?utm_source=github_status&utm_medium=notification) have also added the report command too, please tell if that should be included or not?

Reply:
I considered the `vms/` to be root of project now that you mention it does seem wierd as `vms/vms/` should be the root of project. Changing it in a bit.